ToDoList-Redux/Binders/TaskListViewModel.swift

```diff
@@ -65,25 +57,25 @@ extension Reducer where ActionType == TaskList.Action, StateType == TaskList.Sta
 // MARK: - PROJECTIONS
 
 extension CheckmarkCellView {
-    
     static func viewModel<S: StoreType>(store: S, taskId: String) -> ObservableViewModel<CheckmarkCellView.Action, CheckmarkCellView.State>
-    where S.ActionType == TaskList.Action, S.StateType == TaskList.State {
-        let task = store.mapState { state in state.tasks.first { $0.id == taskId } }
+    where S.ActionType == TaskAction, S.StateType == [Task] {
```

Comment:
Here we are coming from AppState/AppAction objects, instead of coming from View State / View Events. This is because I'm projecting not from the list view, but from outside where I do have the full Store available for me.